File: R/tests/testthat/test_generalise_gs.R
context("Test generalise_gs")
library(hBayesDM)

test_that("Test generalise_gs", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(generalise_gs(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_generalise_gs.py
import pytest

from hbayesdm.models import generalise_gs


def test_generalise_gs():
    _ = generalise_gs(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
